Bottom: f90e95d925322ffa78efdf1ca6a0b64c199f6414
Top:    1162b030446eeebe7335e2f97a5e2b1b90e3de2f
Author: Allen Short <ashort@mozilla.com>
Date:   2019-05-28 15:21:36 -0500

copy extensions into frontend image


---
diff --git a/Dockerfile b/Dockerfile
index 89ee0cdf..6e9d23f3 100644
--- a/Dockerfile
+++ b/Dockerfile
@@ -18,8 +18,7 @@ COPY package.json package-lock.json /frontend/
 RUN npm install
 
 COPY . /frontend
-COPY --from=extension-installer /app/client/app/extensions /app/client/app/extensions
-
+COPY --from=extension-installer /app/client/app/extensions /frontend/client/app/extensions
 RUN npm run build
 
 FROM redash/base:latest
